docker-compose.yaml

```diff
@@ -23,4 +25,20 @@ services:
       foundry:
         condition: service_healthy
     command: "--rpc-url http://foundry:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
-    tty: true
+    tty: true    
+    networks:
+      - devNet
+    
+  dev:
+    container_name: dev
+    build:
+      context: .
+      dockerfile: ./docker-file.dockerfile
+    networks:
+      - devNet    
+    volumes:
+      - ./:/usr/src/builder-examples
```

Comment:
Does this mount the current directory into the path /usr/src/builder-examples?

Reply:
IMO it would be nicer to load it either into builder-examples and skip the usr/src/ or just loading it into the root dir of the build environment